recipes/cuda-cccl/build.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Install to conda style directories
+[[ -d lib64 ]] && mv lib64 lib
+
+[[ ${target_platform} == "linux-64" ]] && targetsDir="targets/x86_64-linux"
+[[ ${target_platform} == "linux-ppc64le" ]] && targetsDir="targets/ppc64le-linux"
+[[ ${target_platform} == "linux-aarch64" ]] && targetsDir="targets/sbsa-linux"
+
+for i in `ls`; do
```

Comment:
This script currently causes `lib/cmake` to be a symlink which will break other conda packages that need to install directories into `lib/cmake` ( Or will fail if that exists as a directory ).

Instead we need to symlink the `lib/cmake/[cub,libcudacxx,thrust]`  directories

Reply:
Thanks Robert! This should be fixed now.

Reply:
Just verified and the update works, thank you